### PR TITLE
Feature PostgreSQL

### PR DESCRIPTION
**Pull request by Mad-Proger**

### CODE REVIEW

**Review by Pelmenner — COMMENTED** [review]
Update the README, please

File: migrations/001_init.sql [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+    internal_id SERIAL PRIMARY KEY,
     CONSTRAINT unique_chat UNIQUE (chat_id, chat_type)
 );
 
 CREATE TABLE IF NOT EXISTS Subscriptions (
-    source_chat INTEGER NOT NULL,
-    destination_chat INTEGER NOT NULL,
-    FOREIGN KEY (source_chat) REFERENCES Chats(rowid),
-    FOREIGN KEY (destination_chat) REFERENCES Chats(rowid),
+    source_chat SERIAL NOT NULL,
```
Comment (Pelmenner) [inline migrations/001_init.sql:11]: SERIAL is already not null. Btw, it should be integer here (and somewhere further) as SERIAL is not an actual datatype

File: orm/orm.go [hunk 2]
```diff
@@ -255,14 +252,15 @@ func GetUnsentMessages(db *sql.DB, maxCnt int) []QueuedMessage {
...
-	row := tx.QueryRow("SELECT rowid FROM Chats WHERE token = $1", &token)
+	row := tx.QueryRow("SELECT internal_id FROM Chats WHERE token = $1", &token)
 	rowID := -1
 	err := row.Scan(&rowID)
 	return rowID, err
 }
 
 // Subscribes proveded chat on another with given token.
-//  Returns true on success
+//
```
Comment (Pelmenner) [inline orm/orm.go:262]: This empty line seems to have no effect

**Review by Pelmenner — APPROVED** [review]

File: orm/orm.go (hunk 2 above)
Comment (Mad-Proger) [reply orm/orm.go:262]: it is added automatically by extension, but no idea why

**Pull request merged**